src/main/java/org/mitre/synthea/export/CSVExporter.java

```diff
@@ -487,7 +487,7 @@ private void medication(String personID, String encounterID,
     s.append(coding.code).append(',');
     s.append(clean(coding.display)).append(',');
 
-    s.append(String.format("%.2f", Costs.calculateCost(medication, true))).append(',');
+    s.append(String.format("%.2f", medication.cost())).append(',');
```

Comment:
Can you add the number of refills and total cost like we talked about?

Reply:
I can, though the numbers will be completely invented, and it will likely use a one-size fits all function that may be completely inappropriate in some cases. 

The MedicationOrder state has an optional `prescription` object that includes # of refills, duration, etc. https://github.com/synthetichealth/synthea/wiki/Generic-Module-Framework%3A-States#prescription But only a few medications have prescription details, and nothing has `refills`. We could calculate something using a `duration` provided, if there is one. But that's completely independent of when medications start and stop so could lead to unusual results. Or I could pick a # of refills assuming 1 per month?

Reply:
Updated with "dispenses" because I wanted it to be clear that the number does include the original prescription. Using the term "refills" makes it less clear as to whether the # represents N or N-1.